src/CouchDB.Driver/CouchDatabase.cs

```diff
@@ -500,6 +500,42 @@ public Task DeleteIndexAsync(IndexInfo indexInfo, CancellationToken cancellation
 
         #endregion
 
+        #region View
+
+        /// <inheritdoc/>
+        public Task<CouchViewResult<TValue>> GetViewAsync<TValue>(string design, string view, CouchViewOptions? options = null, CancellationToken cancellationToken = default)
+        {
+            Check.NotNull(view, nameof(view));
+
+            IFlurlRequest request = NewRequest()
+                .AppendPathSegments("_design", design, "_view", view)
+                .SetQueryParams(options?.ToQueryParameters());
+
+            return request
+                .GetJsonAsync<CouchViewResult<TValue>>(cancellationToken)
+                .SendRequestAsync();
+        }
+
+        /// <inheritdoc/>
+        public Task<CouchViewResult<TValue, TDoc>> GetViewAsync<TValue, TDoc>(string design, string view, CouchViewOptions? options = null, CancellationToken cancellationToken = default)
+            where TDoc : CouchDocument
+        {
+            Check.NotNull(view, nameof(view));
+
+            options ??= new CouchViewOptions();
+            options.IncludeDocs = true;
+
+            IFlurlRequest request = NewRequest()
+                .AppendPathSegments("_design", design, "_view", view)
+                .SetQueryParams(options.ToQueryParameters());
```

Comment:
A utility for query params already exists at line [#L370](https://github.com/matteobortolazzo/couchdb-net/blob/a2f5ad0723ba00f437cb11ccce69ab5ff6f9de91/src/CouchDB.Driver/CouchDatabase.cs#L370)

Reply:
Oh, I didn't see it but does it filter them? I noticed that strings need Quotes and the same for lists etc so I did the extra work instead of letting Flur handle at the time.

Reply:
The current method works so I think it supports everything. Also, now I moved to Flurl 3 that might handle this stuff automatically, but that's a future refactor anyway. Try to give a look at the existing method

Reply:
Hey, I tried using the method but it didn't generate the correct URL for key or keys. Maybe it has something to do with my class. Is it ok to leave it like this for now? And change the name of the method to reflect what it does, maybe even make it an extension method?
I am sure that tests can be written to test if the parameters are correct, eg string requires to have escaped `"` around it, etc.